ablog/about.html

```diff
@@ -2,12 +2,24 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <title>About</title>
+    <title>Aronde | About</title>
+    <link href="style.css" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css?family=Chivo:700|Overpass:400,700&display=swap" rel="stylesheet">
 </head>
 <body>
-<h1>About Aronde</h1>
-<h3> Girl in the wide IT world</h3>
-<p>Lorem ipsum</p>
+<div class="banner">
```

Comment:
Use correct indentation here. You can do it with Reformat command in pycharm: https://www.jetbrains.com/help/pycharm/reformat-and-rearrange-code.html?keymap=primary_windows#reformat_code

Reply:
Do it also for other files so they have consistent indentation.